components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountArabicSnapshotTests.m

```diff
@@ -44,14 +43,9 @@ - (void)setUp {
   self.textFieldController.characterCountViewMode = UITextFieldViewModeAlways;
   MDCSemanticColorScheme *colorScheme =
       [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-  MDCTypographyScheme *typographyScheme =
-      [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
 
   [MDCOutlinedTextFieldColorThemer applySemanticColorScheme:colorScheme
                                       toTextInputController:self.textFieldController];
-  [MDCTextFieldTypographyThemer applyTypographyScheme:typographyScheme
-                                toTextInputController:self.textFieldController];
-  [MDCTextFieldTypographyThemer applyTypographyScheme:typographyScheme toTextInput:self.textField];
```

Comment:
Aren't you missing a call to theme self.textField?
The theming extension seem to applySemanticColorScheme only to self.textFieldController, not to self.textField, and themers seem to apply theming to both.

Reply:
There is no theming extension for MDCTextField so I'm only using the theming extension on the controllers.

Reply:
I think the controllers all have access to their textfields via the `.textInput` property.

Reply:
They do, but I didn't want to modify them in this PR.